lib/clowder-common-ruby/types.rb

```diff
@@ -56,6 +57,7 @@ def valid_keys
         keys << :endpoints
         keys << :privateEndpoints
         keys << :BOPURL
+        keys << :hashCache
```

Comment:
Just curious, what's the purpose of this?

Reply:
`hashCache` was introduced in https://github.com/RedHatInsights/clowder/pull/776 but to be honest I have no idea what is its purpose without studying the clowder codebase.